### PR TITLE
fix: stop displaying empty descriptions

### PR DESCRIPTION
**Pull request by olga-union**
Signed-off-by: Olga Nad <olga@union.ai>

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Workflow and task descriptions aren't properly populating due to issues on the backend, while they are being addressed, empty description string should not be rendered.

## Demo
Workflows
Screen Shot 2022-03-25 at 9 56 39 AM
Tasks
Screen Shot 2022-03-25 at 9 56 52 AM

### CODE REVIEW

**Review by anrusina — DISMISSED** [review]
LGTM, with small comment

File: src/components/Task/SearchableTaskNameList.tsx [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 import ChevronRight from '@material-ui/icons/ChevronRight';
 import ErrorOutline from '@material-ui/icons/ErrorOutline';
 import classnames from 'classnames';
-import { noDescriptionString } from 'common/constants';
```
[the remark this answers is not in the document]
Comment (olga-union) [reply src/components/Task/SearchableTaskNameList.tsx:6]: @anrusina done!

**Review by anrusina — APPROVED** [review]
Lgtm

**Comment by jsonporter** [comment]
lgtm

**Pull request merged**